array-manipulation-1/src/reverse.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable no-unused-vars */
+function reverse(array) {
+  var result = [];
+  var last = array.length - 1;
+  for (var i = last; i >= 0;) {
+    result.push(array[i]);
+    i--;
```

Comment:
Why is this not in the Final Expression section of your loop config? 🤔

Reply:
I think because I put it there before and the code didn't work.... But... now that I think about it, maybe I didn't do the condition correctly before. I'll fix it right away!